### ISSUE TITLE
Support for seq2seq models 

### ISSUE THREAD

**Issue by maximus-21** [EricLBuehler/xlora#29]
I was trying to use xlora for combining Flan-T5 LoRAs and ran into error within apply_scalings_to_x, does xLoRA support seq2seq models such as Flan-T5 and BART ?

**Comment by yuxiang-guo** [comment]
> I was trying to use xlora for combining Flan-T5 LoRAs and ran into error within apply_scalings_to_x, does xLoRA support seq2seq models such as Flan-T5 and BART ?

Hi, I also want to apply xlora to T5, did you try to do that?

**Comment by maximus-21** [comment]
> > I was trying to use xlora for combining Flan-T5 LoRAs and ran into error within apply_scalings_to_x, does xLoRA support seq2seq models such as Flan-T5 and BART ?
> 
> Hi, I also want to apply xlora to T5, did you try to do that?
>
>Yeah I did, it wasn't working. I came across a tensor related error when doing the forward pass

**Comment by maximus-21** [comment]
@yuxiang-guo Yeah I did. But I faced a tensor related error while doing the forward pass. I tried debugging through it but couldn't exactly understand what was the problem here.
